CHANGELOG.md

```diff
@@ -500,3 +500,6 @@ Released with 1.0.0-beta.37 code base.
 -  Added effectiveGasPrice to TransactionReceipt (#4692)
 -  Correction in documentation for `web3.eth.accounts.signTransaction` (#4576)
 -  Updated README to include Webpack 5 create-react-app support instructions (#4173)
+
+### Changed
+-  Muted E2E gnosis dex tests in CI (#4436)
```

Comment:
Would be nice to mention the reason of muting these tests with some reference issue to fix and unmute in the future.